blog/2020-11-19-error-message-design.md

```diff
@@ -0,0 +1,62 @@
+---
+tags:
+- developer-experience
+- errors
+posted_on_: 2020-11-19T23:06:09Z
+id: error-message-design
+title: Write Errors That Don't Make Me Think
+author: Swyx (Shawn Wang)
+author_title: Product Manager
+author_image_url: https://avatars2.githubusercontent.com/u/6764957?s=460&u=97ad815028595b73b06ee4b0510e66bbe391228d&v=4
+release_version: V1.3
+
+---
+
```

Comment:
You're going to need a truncate tag or the entire blog content will show up in the feed.

```suggestion
<!--truncate-->
```